packages/next/test/fixtures/00-server-build/index.test.js

```diff
@@ -22,7 +22,7 @@ async function checkForChange(url, initialValue, getNewValue) {
 const ctx = {};
 
 describe(`${__dirname.split(path.sep).pop()}`, () => {
-  it('should deploy and pass probe checks', async () => {
```

Comment:
Was this change intentional?

Reply:
Yes. It cleans up some test pollution I noticed while running these tests to make sure changes didn't break anything.

Reply:
if you change this to `beforeAll()` and it fails the tests will hang indefinitely

Reply:
@ijjk I simulated an error: [`c9c4225` (#9196)](https://github.com/vercel/vercel/pull/9196/commits/c9c4225361fe63bdc1e9cf647fab5361797256e8)

and it seemed to fail fine: https://github.com/vercel/vercel/actions/runs/4105864790/jobs/7083376282#step:9:465

but I'm happy to revert if this is an issue.

Reply:
reverted for now: [`ba22f16` (#9196)](https://github.com/vercel/vercel/pull/9196/commits/ba22f16676c2a62ad9f8e280911973ef063821b5)